### DISCUSSION TITLE
Why are there some red crosses in nvim-tree

### DISCUSSION THREAD

**Discussion by zhuoqun-chen** [nvim-tree/nvim-tree.lua#1748] [General]
Sorry about this silly question, I'm new to nvim and I found this plugin to show me my file tree. My question is as the title, Why are there some red crosses in nvim-tree? I've set those config `.lua` correctly.

<img width="1477" alt="img-20221115_211859" src="https://user-images.githubusercontent.com/73287160/202090132-9426f5ac-218e-4541-9d7c-4e5327cbcabb.png">

**Comment by zhuoqun-chen** [comment]
Does it mean that these files are not staged and committed yet? I seem to understand.

**Reply by alex-courtis** [reply]
Yes. The `renderer.icons.glyphs.git.unstaged` icon is for "dirty" files:

Unstaged:
* modified
* copied
* type changed
